### PR TITLE
My branch

### PR DESCRIPTION
**Pull request by nilab-khwaja**
first microvers project for testing linter

### CODE REVIEW

**Comment by nilab-khwaja** [comment]
first project 'Hello-Microvers' project using html and css and adding linter and readme file

File: README.md [hunk 1]
```diff
@@ -1 +1,162 @@
...
+
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
```
Comment (GedeonTS) [inline README.md]: - Kindly remove the micro verse logo to make your project look professional :recycle:

File: README.md [hunk 2]
```diff
@@ -1 +1,162 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+
+
+> Write a message to encourage readers to support your pro
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
```
Comment (GedeonTS) [inline README.md]: - Kindly customize your Acknowledgment section to make it look more professional:recycle:

**Review by GedeonTS — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED :recycle: :

Hi @nilab-khwaja , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights :tophat:  :+1: 
- No linter errors :+1: 
- Good use of GitHub flow :+1: 

## Required Changes ♻️
- Kindly make sure test.md is not seen in GitHub :recycle: 
_Check the comments under the review._

## Optional suggestions
- [OPTIONAL] It would be better to start your commit messges with a capital letter in order to make it look more professiona; :recycle: 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by nilab-khwaja** [comment]
Update .gitignore file to not visible on the branch.

File: README.md [hunk 3]
```diff
@@ -1 +1,162 @@
-# hello-microvers
\ No newline at end of file
+# hello-microvers
+<a name="readme-top"></a>
+
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
```
Comment (emmiiorji) [inline README.md:7]: You seemed to have forgotten to remove the Microverse logo as requested by the previous reviewer. Kindly do so or replace it with your personal logo.

**Review by emmiiorji — APPROVED** [review]
.

File: README.md (hunk 3 above)
Comment (emmiiorji) [inline README.md]: You seemed to have forgotten to remove the Microverse logo as requested by the previous reviewer. Kindly do so or replace it with your personal logo.

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @nilab-khwaja, 
# Status: Changes Required ♻️

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

BbcanBbcan3GIF

## Good Highlights✔️

- The test.md file is properly ignored

## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by nilab-khwaja** [comment]
README.md file updated.

**Review by wayungi — APPROVED** [review]
Hi @nilab-khwaja , 
image

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- [ ] All linter checks passing ✔️ 
- [ ] Good commit messages ✔
- [ ] No merge conflicts ✔️ 
- [ ]  Correct GitHub flow ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**